web/styles/embed/styles_dark.scss

```diff
@@ -35,8 +35,14 @@ a {
   background-color: $dark-code-background-color;
 }
 
-#console-output-container span.normal {
-  color: #dddddd;
+#console-output-container {
+  span.normal {
+    color: #dddddd;
+  }
+
+  .error-output {
```

Comment:
This will close an additional bug, won't it? If so, it would be good to list it in the PR description as well.

Reply:
ah yeah this also fixes an issue where the output was difficult to see (for some reason the console output was showing up as grey on black in dark mode)

Reply:
I don't think there's an issue for it, just something I noticed.